client/src/components/PageComponents/ListContainer/ListContainer.tsx

```diff
@@ -3,26 +3,24 @@ import List from "../List/List";
 import { IoArrowBackCircleOutline, IoArrowForwardCircleOutline } from "react-icons/io5";
 import { arrowIcon, dailyTaskContainer, divArrow } from "./ListContainer.css";
 
-const ListContainer : FC = () => {
-    const temp = [1, 2, 3, 4];
+const ListContainer: FC = () => {
+  const temp = [1, 2, 3, 4];
 
-    return (
-        <div>
-            <div className={dailyTaskContainer}>
-                <div className={divArrow}>
-                    <IoArrowBackCircleOutline className={arrowIcon} />
-                </div>
-                {
-                    temp.map(() => (
-                        <List />
-                    ))
-                }
-                <div className={divArrow}>
-                    <IoArrowForwardCircleOutline className={arrowIcon} />
-                </div>
-            </div>
+  return (
+    <div>
+      <div className={dailyTaskContainer}>
+        <div className={divArrow}>
+          <IoArrowBackCircleOutline className={arrowIcon} />
         </div>
-    );
-}
+        {temp.map(() => (
+          <List />
+        ))}
```

Comment:
map을 돌릴때 콜백함수를 사용하지 않고 이렇게 사용하는건 의도와 다르게 동작하는거라 좋지 않은데요.
여기서는 아래와 같이 표현해주면 될 것 같아요.
```suggestion
        <List />
        <List />
        <List />
        <List />
        ))}
```